src/main/java/bookstore/controller/BookController.java

```diff
@@ -36,13 +37,14 @@ public BookDto getBookById(@PathVariable Long id) {
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
-    public BookDto createBook(@RequestBody CreateBookRequestDto book) {
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto book) {
         return bookService.save(book);
     }
 
     @PutMapping("/{id}")
     @ResponseStatus(HttpStatus.OK)
-    public BookDto updateBookById(@PathVariable Long id, @RequestBody CreateBookRequestDto book) {
+    public BookDto updateBookById(@PathVariable Long id,
```

Comment:
you can also validate `@PathVariable` like:
```
@PathVariable @Positive Long id
```

in order to do so - annotate you controller with `@Validated`